commcare_connect/opportunity/models.py

```diff
@@ -263,7 +263,9 @@ def learn_progress(self):
 
     @property
     def visit_count(self):
-        return self.completedwork_set.exclude(status=CompletedWorkStatus.over_limit).count()
+        return sum(
```

Comment:
Do we want to exclude duplicates here? I forget a bit what we decided in the past.

Reply:
We did not exclude duplicates in the older code also. I am not sure what we decided for the duplicates.